packages/puppeteer-core/package.json

```diff
@@ -108,6 +108,7 @@
   "files": [
     "lib",
     "src",
+    "!*.test.ts",
```

Comment:
Should we just ignore everything that has `.test.` as in this glob `!*.test.*` ?

Reply:
We could, not sure if we will eventually face the opposite issue.